packages/orchestration/src/exos/chain-hub.js

```diff
@@ -41,13 +41,25 @@ export const DenomDetailShape = M.splitRecord(
   { brand: BrandShape },
 );
 
-// TODO refactor into an enum-ish object
-/** agoricNames key for ChainInfo hub */
-export const CHAIN_KEY = 'chain';
-/** namehub for connection info */
-export const CONNECTIONS_KEY = 'chainConnection';
-/** namehub for assets info */
-export const ASSETS_KEY = 'chainAssets';
+/**
+ * @enum {(typeof HubName)[keyof typeof HubName]}
+ */
+export const HubName = /** @type {const} */ ({
+  /** agoricNames key for ChainInfo hub */
+  Chain: 'chain',
+  /** namehub for connection info */
+  ChainAssets: 'chainAssets',
+  /** namehub for assets info */
+  ChainConnection: 'chainConnection',
```

Comment:
```suggestion
  /** namehub for assets info */
  ChainAssets: 'chainAssets',
  /** namehub for connection info */
  ChainConnection: 'chainConnection',
```

Also consider using singular and plural variables consistently - I think these are both one to many.

Reply:
These names just match the vstorage keys and changing the keys in vstorage would be a breaking change. We don't have anything in production yet so I'd be open to it but out of scope of this PR. I think something for the tickets about nailing down the vstorage schema for chain info.